app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.name = "actors"
+
+    def all(self) -> None:
```

Comment:
wrong return annotation
```suggestion
    def all(self) -> [Actor]:
```